examples/README.md

```diff
@@ -2,19 +2,18 @@
 
 Consists of various examples to get familiar with writing GTK applications using the Rust bindings.
 
-- [Custom Application](./examples/src/bin/application_subclass.rs)
-- [Basic Sample](./examples/src/bin/basic.rs)
-- [GtkBuilder Basics Sample](./examples/src/bin/builder_basics.rs)
-- [Using the Builder pattern](./examples/src/bin/builders.rs)
-- [Clock Sample](./examples/src/bin/clock.rs)
-- [Composite Template](./examples/src/bin/composite_template.rs)
-- [CSS](./examples/src/bin/css.rs)
-- [Custom Editable](./examples/src/bin/custom_editable.rs)
-- [Custom Paintable](./examples/src/bin/custom_paintable.rs)
-- [Dialog](./examples/src/bin/dialog.rs)
-- [Entry Completion](./examples/src/bin/entry_completion.rs)
-- [Entry Undo](./examples/src/bin/entry_undo.rs)
-- [Orientable Widget](./examples/src/bin/orientable_subclass.rs)
-- [TextView](./examples/src/bin/text_viewer.rs)
-- [Video Player](./examples/src/bin/video_player.rs)
-- [Custom Widget](./examples/src/bin/widget_subclass.rs)
+- [Basic example](./basics/main.rs)
+- [Using the Builder pattern](./builder_pattern/main.rs)
+- [Clock example](./clock/main.rs)
+- [Composite Template](./composite_template/main.rs)
+- [CSS](./css/main.rs)
+- [Custom Editable](./custom_editable/main.rs)
+- [Custom Orientable](./custom_orientable/main.rs)
+- [Custom Paintable](./custom_paintable/main.rs)
+- [Custom Widget](./custom_widget/main.rs)
+- [Dialog](./dialog/main.rs)
+- [Entry Completion](./entry_completion/main.rs)
+- [Entry Undo](./entry_undo/main.rs)
+- [GtkBuilder example](./gtk_builder/main.rs)
+- [TextView](./text_viewer/main.rs)
+- [Video Player](./video_player/main.rs)
```

Comment:
```suggestion
- [Basic example](./basics/)
- [Using the Builder pattern](./builder_pattern/)
- [Clock example](./clock/)
- [Composite Template](./composite_template/)
- [CSS](./css/)
- [Custom Editable](./custom_editable/)
- [Custom Orientable](./custom_orientable/)
- [Custom Paintable](./custom_paintable/)
- [Custom Widget](./custom_widget/)
- [Dialog](./dialog/)
- [Entry Completion](./entry_completion/)
- [Entry Undo](./entry_undo/)
- [GtkBuilder example](./gtk_builder/)
- [TextView](./text_viewer/)
- [Video Player](./video_player/)
```
No need to point to the source code, there are README's for each example now

Reply:
I was wondering which would better, if you are more sure let's point to the READMEs :)